nucliadb_sdk/nucliadb_sdk/tests/test_search_resource.py

```diff
@@ -269,7 +269,7 @@ def test_search_resource(knowledgebox: KnowledgeBox):
     resources = knowledgebox.search(
         vector=vector_q, vectorset="all-MiniLM-L6-v2", min_score=0.70
     )
-    assert len(resources.sentences.results) == 11
+    assert len(resources.sentences.results) == 20
```

Comment:
Why the number of results change all of a sudden ?

Reply:
Because of this fix from @hermeGarcia https://github.com/nuclia/nucliadb/commit/53bba67bd6a47e22bdfd8ba994f92ca4f1af0f38

Reply:
Make sense :cat: